sql/src/test/resources/calcite/tests/window/wikipediaAggregationsMultipleOrderingDesc.sqlTest

```diff
@@ -10,16 +10,22 @@ sql: |
     FROM wikipedia
     GROUP BY 1, 2
     ORDER BY 1 DESC, 2 DESC
+    LIMIT 2
 
 expectedOperators:
+  - {type: "naiveSort", columns: [{column: "d0", direction: "ASC"}, {column: "d1", direction: "DESC"}]}
   - { type: "naivePartition",  partitionColumns: [ "d0" ] }
   - type: "window"
     processor:
       type: "framedAgg"
       frame: { peerType: "ROWS", lowUnbounded: false, lowOffset: 3, uppUnbounded: false, uppOffset: 2 }
       aggregations:
         - { type: "longSum", name: "w0", fieldName: "a0" }
-  - { type: "naiveSort", columns: [ { column: "d1", direction: "DESC" }, { column: "a0", direction: "DESC"} ]}
```

Comment:
Unclear as in why this changed from ASC to DESC. Investigating further

Reply:
any findings?